docs/faq.rst

```diff
@@ -150,7 +150,11 @@ My machine goes to sleep causing `Broken pipe`
 ----------------------------------------------
 
 While backing up your data over the network, your machine should not go to sleep.
-On macOS you can use `caffeinate` to avoid that.
+On Linux you can use `systemd-inhibit` to avoid that. On macOS you can use `caffeinate`.
+
+``systemd-inhibit borg create repo::archive path``
+
+``caffeinate -i borg create repo::archive path``
```

Comment:
thanks for the PR!

the syntax you used here is for borg 1.x, but this is master branch (borg 2).

maybe just use "borg create ..." - that is correct for both versions.

later, a backport of this to 1.4-maint branch will be needed, so it can go into 1.4.2 release.